src/eko/evolution_operator/__init__.py

```diff
@@ -286,10 +286,12 @@ def grid_size(self):
     def a_s(self):
         """Returns the computed values for :math:`a_s`"""
         sc = self.managers["strong_coupling"]
-        a0 = sc.a_s(
+        a0 = sc.a(
```

Comment:
Maybe here and below I'd rename variables to `as0` and `as1`, such that it is clear at first inspection that we are still talking about the strong coupling.